### ISSUE TITLE
r_band mean magnitude around redmapper and random sample

### ISSUE THREAD

**Issue by WentaoLuo** [WentaoLuo/Galaxy_Galaxy_Magnification#3]
![random_r_band_mean_magnitude](https://cloud.githubusercontent.com/assets/6350744/14358780/85190bd8-fcbc-11e5-809b-076a46f10f80.png)
![redmapper_r_band_mean_magnitude](https://cloud.githubusercontent.com/assets/6350744/14358783/8abd657a-fcbc-11e5-967c-2d224d238e07.png)

The top figure is measured around random sample and the bottom one is around redmapper clusters.

**Comment by WentaoLuo** [comment]
The difference of the two measurements is caused by both magnification and
reddening as in Menard et al 2007. But they use angular coordinates and quasar sample.
Whereas we use background galaxies as a whole.

**Comment by WentaoLuo** [comment]
![random_g_r_color](https://cloud.githubusercontent.com/assets/6350744/14359056/b97dddd0-fcbd-11e5-8a82-5328d57f5808.png)
![redmapper_g_r_color](https://cloud.githubusercontent.com/assets/6350744/14359057/bbd31af0-fcbd-11e5-9459-d9d96eac245e.png)

I just do not understand what is going on with the color around redmapper.
At smaller scale and larger scale.

**Comment by WentaoLuo** [comment]
Any suggestions?

**Comment by WentaoLuo** [comment]
![rmean_plot](https://cloud.githubusercontent.com/assets/6350744/14392441/5b03f596-fd8f-11e5-92a8-4e1a9ac06521.png)
![gmean_plot](https://cloud.githubusercontent.com/assets/6350744/14392452/674f0ed0-fd8f-11e5-8b44-20a353416237.png)
![color_plots](https://cloud.githubusercontent.com/assets/6350744/14392458/6f4f60e4-fd8f-11e5-8ded-406c35772127.png)

![zmean_plot](https://cloud.githubusercontent.com/assets/6350744/14392463/7379ac1a-fd8f-11e5-998e-377c3f8b3ecd.png)

Updated plots

**Comment by lidenscheng** [comment]
Is this still random sample vs redmapper clusters?

On Fri, Apr 8, 2016 at 10:40 AM, Wentao Luo notifications@github.com
wrote:

> [image: rmean_plot]
> https://cloud.githubusercontent.com/assets/6350744/14392441/5b03f596-fd8f-11e5-92a8-4e1a9ac06521.png
> [image: gmean_plot]
> https://cloud.githubusercontent.com/assets/6350744/14392452/674f0ed0-fd8f-11e5-8b44-20a353416237.png
> [image: color_plots]
> https://cloud.githubusercontent.com/assets/6350744/14392458/6f4f60e4-fd8f-11e5-8ded-406c35772127.png
> 
> [image: zmean_plot]
> https://cloud.githubusercontent.com/assets/6350744/14392463/7379ac1a-fd8f-11e5-998e-377c3f8b3ecd.png
> 
> Updated plots
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> https://github.com/WentaoLuo/Galaxy_Galaxy_Magnification/issues/3#issuecomment-207532289

## 

Lidens
[image: Namez] http://namez.com/profiles/2398-lidens-cheng/?autoPlay=true

**Comment by WentaoLuo** [comment]
Yes,m Sukhdeep suggests that we'd better put the random catalog and redmapper 
together so I put them together so that we could clearly tell the difference fromd the two samples.

**Comment by lidenscheng** [comment]
I have a redmapper expert here in my building, Eduardo Rozo that is. Do you
want me to ask him what could be going on?

On Friday, April 8, 2016, Wentao Luo notifications@github.com wrote:

> Yes,m Sukhdeep suggests that we'd better put the random catalog and
> redmapper
> together so I put them together so that we could clearly tell the
> difference fromd the two samples.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> https://github.com/WentaoLuo/Galaxy_Galaxy_Magnification/issues/3#issuecomment-207685251

## 

Lidens
[image: Namez] http://namez.com/profiles/2398-lidens-cheng/?autoPlay=true

**Comment by WentaoLuo** [comment]
That will be great. Thanks Lidens. He is one of the creator of redmapper catalog.
So my questions are as follows:
1, How much is the contamination from member galaxies, which are not identified in redmapper due to 
    photoZ. This could also cause the "magnification" as systematics.
2, If the color as a function of radius is real, why there the first two points are bluer?

**Comment by WentaoLuo** [comment]
And we could first use illustris to estimate the magnification, shear, kappa and reddening.
Nan Li has provided the density, kappa and shear estimator. So we could apply this 
code to create magnification bias first using the luminosity function of galaxies at high z snap shots.

**Comment by WentaoLuo** [comment]
It seems that this project needs more time and detailed evaluation.

**Comment by WentaoLuo** [comment]
I will definitely continue this lovely project.

**Comment by lidenscheng** [comment]
Okay I talked to Eduardo. He said not to trust points from r<200 kpc/h because they're likely garbage. In that region, you're just testing how good you're deblender is. Central galaxy can be very big and anyway don't go lower than r <200 kpc/h.

**Comment by WentaoLuo** [comment]
Thanks Lidens. I think we still get the difference above 200kpc/h.  I will update the results tomorrow
after I finish my talk at UMass.
